chess2.cpp

```diff
@@ -0,0 +1,882 @@
+#include "chess2.h"
```

Comment:
Resubmit with following Code  Guidelines:

1. Function names, Function paramaters should use camel case  eg: performAction .
2. Class Name should use Upper Camel Case (ClassClass)
3. Code should be formated using google c++ formatter